tests/end_to_end/test_vulnerability_detector/test_vulnerability_detector.py

```diff
@@ -47,6 +43,7 @@
 import time
 import ast
 import datetime
+from typing import Generator
```

Comment:
This caused problems for @rafabailon in [this build](https://ci.wazuh.info/job/Test_e2e_system/235/console), it should be reviewed.

Reply:
This fixes the error in @rafabailon build. It was never addressed in any separated issue.
I suggest include the fix here. I include it in the issue description